### PR TITLE
Use patched version of @nimiq/core-web to force JS worker on iOS

### PR DESCRIPTION
**Pull request by sisou**

### CODE REVIEW

File: src/common.js [hunk 1]
```diff
@@ -16,11 +16,9 @@ let __nimiqLoaded = null;
 if ((BrowserDetection.isIOS() || BrowserDetection.isSafari()) && 'serviceWorker' in navigator) {
     // Register service worker to strip cookie from requests.
     // This file is always called from a ./request/*/ folder, hence the paths.
-    navigator.serviceWorker.register('../../ServiceWorker.js', {
-        scope: '../../',
-    }).then(reg => {
+    navigator.serviceWorker.register('/ServiceWorker.js').then(reg => {
```
Comment (nibhar) [inline src/common.js]: Screen Shot 2019-05-21 at 13 10 52
I don't think that works as intended.

Comment (nibhar) [reply src/common.js]:
```suggestion
    navigator.serviceWorker.register('/src/ServiceWorker.js').then(reg => {
```

Comment (sisou) [reply src/common.js]: ah, it fails locally... :(

Comment (sisou) [reply src/common.js]: Ok, I'll remove the registration adaptions

File: src/common.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ async function loadNimiq() {
     // eslint-disable-next-line no-return-assign
     return __nimiqLoaded || (__nimiqLoaded = new Promise(async resolve => {
         // Load web assembly encryption library into browser (if supported)
-        await Nimiq.WasmHelper.doImport();
+        await Nimiq.WasmHelper.doImport(BrowserDetection.isIOS());
```
Comment (Bettelstab) [inline src/common.js]:
```suggestion
        await Nimiq.WasmHelper.doImport(BrowserDetection.isIOS() && !window.opener);
```

**Pull request merged**